Makefile

```diff
@@ -50,30 +70,32 @@ test_e2e_morse_relay: ## Run an E2E Morse relay test
-	@if [ ! -f ./config/.config.yaml ]; then \
-		cp ./cmd/.config.shannon_example.yaml ./config/.config.yaml; \
-		echo "###########################################################################################################################"; \
-		echo "### Created ./config/.config.yaml                                                                                       ###"; \
-		echo "### README: Please update the the following in .config.yaml: 'gateway_private_key_hex' & 'owned_apps_private_keys_hex'. ###"; \
-		echo "###########################################################################################################################"; \
+	@if [ ! -f ./bin/config/.config.yaml ]; then \
+		mkdir -p bin/config; \
+		cp ./cmd/.config.shannon_example.yaml ./bin/config/.config.yaml; \
+		echo "#######################################################################################################"; \
+		echo "### Created ./bin/config/.config.yaml                                                               ###"; \
+		echo "### README: Please update the the following in .config.yaml: gateway_private_key & gateway_address. ###"; \
+		echo "#######################################################################################################"; \
 	else \
-		echo "##############################################################"; \
-		echo "### ./config/.config.yaml already exists, not overwriting. ###"; \
-		echo "##############################################################"; \
+		echo "##################################################################"; \
+		echo "### ./bin/config/.config.yaml already exists, not overwriting. ###"; \
+		echo "##################################################################"; \
 	fi
 
 .PHONY: copy_morse_config
 copy_morse_config: ## copies the example morse configuration yaml file to .config.yaml file
-	@if [ ! -f ./config/.config.yaml ]; then \
-		cp ./cmd/.config.morse_example.yaml ./config/.config.yaml; \
-		echo "#############################################################################################################"; \
-		echo "### Created ./config/.config.yaml                                                                         ###"; \
-		echo "### README: Please update the the following in .config.yaml: 'url', 'relay_signing_key', & 'signed_aats'. ###"; \
-		echo "#############################################################################################################"; \
+	@if [ ! -f ./bin/config/.config.yaml ]; then \
+		mkdir -p bin/config; \
+		cp ./cmd/.config.morse_example.yaml ./bin/config/.config.yaml; \
+		echo "#######################################################################################################"; \
+		echo "### Created ./bin/config/.config.yaml                                                               ###"; \
+		echo "### README: Please update the the following in .config.yaml: gateway_private_key & gateway_address. ###"; \
```

Comment:
This should refer to the morse-specific variables: `relay_signing_key`, `signed_aats`, etc.